video/out/vo_gpu_next.c

```diff
@@ -1221,6 +1221,7 @@ static int reconfig(struct vo *vo, struct mp_image_params *params)
         return -1;
 
     resize(vo);
+    TA_FREEP(&vo->target_params);
```

Comment:
If it is not cleared externally `target_params` can be put into `priv` struct.

Reply:
but it's used externally?

Reply:
it's used `vo->target_params` is used externally, but we don't need to dynamically allocate it anymore I guess and do similarly as you did for other vo. And just during runtime it would update parameters.

Reply:
I get what you meant now.
But it's reasonably correct that `target_params` is unset after a reconfig happens before the next frame is drawn (since gpu-next has dynamic hdr metadata and stuff), so I don't think it's totally necessary to refactor this.

Reply:
ok